discordia/containers/snowflakes/channels/TextChannel.lua

```diff
@@ -101,6 +101,15 @@ local function sendMessage(self, content, mentions, tts)
 	if success then return self._messages:new(data) end
 end
 
+local function sendEmbed(self, embed, content)
```

Comment:
I need to rework message sending. Embed sending will be integrated into existing `sendMessage` methods.